### PR TITLE
add community link to the navbar

### LINKED ISSUE #352: Adding Community link to user guide
Hi CWL  team,

It would be great addition to the user guide to add link to community meetings/chat rooms [in the header at the top], as most users spend there time going through it. : )

### PR DESCRIPTION
**Pull request by Mackenzie-OO7**
resolves #352

### CODE REVIEW

**Comment by Mackenzie-OO7** [comment]
@kinow The current theme displays the last item on the navbar as a dropdown; if they exceed 5, do you know if there's a way to change that?
Screenshot 2022-12-27 131713

File: src/index.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ topics/index.md
 tutorials.md
 faq.md
 LICENSE.md
+Community.md
```
Comment (kinow) [inline src/index.md:39]: Might be better to use `community.md` to keep things homogeneous, as other files like `tutorials.md` and `faq.md` also start with lower case letter.

**Comment by kinow** [comment]
> @kinow The current theme displays the last item on the navbar as a dropdown; if they exceed 5, do you know if there's a way to change that? Screenshot 2022-12-27 131713

We use the PyData theme. They have a documentation where most (if not all) options available are listed with some explanation.

For what you are looking for, I think it'd be this one: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/header-links.html#navigation-bar-dropdown-links :+1: 

Thanks for the PR @Mackenzie-OO7 !

**Comment by Mackenzie-OO7** [comment]
> > @kinow The current theme displays the last item on the navbar as a dropdown; if they exceed 5, do you know if there's a way to change that? Screenshot 2022-12-27 131713
> 
> We use the PyData theme. They have a documentation where most (if not all) options available are listed with some explanation.
> 
> For what you are looking for, I think it'd be this one: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/header-links.html#navigation-bar-dropdown-links 👍
> 
> Thanks for the PR @Mackenzie-OO7 !

Thank you!

**Review by mr-c — CHANGES_REQUESTED** [review]
Hello @Mackenzie-OO7 ! Thanks for doing this. Instead of redirecting from a new (empty) page, can you instead use <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/header-links.html#navigation-bar-external-links> by adding a `external_links` section to the `html_theme_options` section in `src/conf.py`?

Thanks

**Review by mr-c — APPROVED** [review]

**Pull request merged**